services/movie.py

```diff
@@ -0,0 +1,40 @@
+from django.db.models import QuerySet, Q
+from db.models import Movie, Genre, Actor
+
+
+def get_movies(
+    genres_ids: list[int] = None,
+    actors_ids: list[int] = None
+) -> QuerySet | Movie:
+    queryset = Movie.objects.all()
+    if genres_ids is None and actors_ids is None:

```

Comment:
Here you can basically remove lines 10-15. Keep `if genres_ids` and `if actors_ids` but do not `return` in each `if` statement - return filtered queryset only once in the end